torchtune/_cli/cp.py

```diff
@@ -66,7 +66,7 @@ def main(parser):
         epilog=textwrap.dedent(
             """\
         examples:
-            $ tune cp lora_finetune_distributed.yaml ./my_custom_llama2_lora.yaml
+            $ tune cp llama2/7B_lora.yaml ./my_custom_llama2_lora.yaml
```

Comment:
@joecummings I think appending the model name (eg: llama2) is fine and in fact will be critical as we add more models. I tested and this should work with our pkg structure. Let me know what you think.